hello-world-master/source/components/scene/helloworld.brs

```diff
@@ -1,25 +1,20 @@
 sub init()
     m.imageRowList = m.top.findNode("imageRowList")
     m.button = m.top.findNode("button")
-    showImage()
     m.imageRowList.setFocus(true)
     m.button.observeField("buttonSelected", "onButtonSelected")
     m.imageRowList.observeField("rowItemSelected", "onRowItemSelected")
+    m.getRequestTask = CreateObject("roSGNode", "GetRequestTask")
+    m.getRequestTask.ObserveField("itemContent", "onFetchData")
+	  m.getRequestTask.control = "RUN"
 end sub
 
-sub showImage()
-    listRoot = CreateObject("roSGNode","ContentNode")
-    row = CreateObject("roSGNode","ContentNode")
-    for i = 1 to 5 
-        rowChild = CreateObject("roSGNode","ContentNode")
-        rowChild.FHDPosterUrl = Substitute("pkg:/images/item{0}.jpg",i.toStr())
-        row.appendChild(rowChild)
-    end for
-    
-    listRoot.appendChild(row)
-    m.imageRowList.content = listRoot
+
+sub onFetchData() as void
+  m.imageRowList.content =  m.getRequestTask.itemcontent
```

Comment:
Could we try to update the function signature to include a parameter and use that instead of `m.getRequestTask.itemcontent`?